app/assets/javascripts/discourse/app/templates/modal/create-account.hbs

```diff
@@ -22,7 +17,16 @@
                 </div>
               {{/if}}
               <div class="input-group create-account-email">
-                {{input type="email" disabled=emailDisabled value=accountEmail id="new-account-email" name="email" class=(value-entered accountEmail) autofocus="autofocus" focusOut=(action "checkEmailAvailability")}}
+                {{input type="email"
+                        disabled=emailDisabled
+                        value=accountEmail
+                        id="new-account-email"
+                        name="email"
+                        class=(value-entered accountEmail)
+                        autofocus="autofocus"
+                        focusOut=(action "checkEmailAvailability")
+                        aria-describedby="account-email-validation"
+                        aria-invalid=emailValidation.failed}}
```

Comment:
I don't want to annoy you too much with this but FTR, proper formating is:


```
{{input
  foo=bar
  baz=fooz
}}
``

Reply:
Ah thanks I wasn't sure. You don't have a tool to autoformat these do you?

Reply:
I do have one :)

```
find . -name '*.hbs' | xargs yarn prettier --write --parser=glimmer
```
Now I see your question, why dont we use it? Well we can talk of it on chat if you want ;)